### DISCUSSION TITLE
How to interact with the robot in the input box?

### DISCUSSION THREAD

**Discussion by dclhaha** [telegraf/telegraf#1885] [Q&A]
Like this

https://github.com/telegraf/telegraf/assets/118969145/8c627ddb-ffd4-44c0-8fd9-4441178c92b1

What API can this be implemented through?

**Comment by MKRhere** [comment]
You should enable inline mode in Botfather, and listen for inline_query updates. [Example](https://github.com/feathers-studio/telegraf-docs/blob/b694bcc36b4f71fb1cd650a345c2009ab4d2a2a5/examples/inline-bot.ts).
